docs/dns-configuration.md

```diff
@@ -7,6 +7,8 @@ setting. You'll be able to view all your existing domains here, as well as add n
 
 ![Screenshot of the Domains settings panel](./assets/domain-list.png)
 
+> **Note:** If you’re unable to add or modify domains in the Altis Dashboard settings panel, it’s likely that self-service domain management is disabled for your environment. In this case, please [contact Altis support](support://new) to request domain changes or enable this feature.
```

Comment:
As per our standards, could you wrap the text at 132 columns, please. 
When working on docs, you can run the docs linter https://docs.altis-dxp.com/dev-tools/linting-your-documents/